<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdBlockCreateFilterWidget</class>
 <widget class="QWidget" name="AdBlockCreateFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filters:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="blockingFilter">
       <property name="text">
        <string>Blocking filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="exceptionFilter">
       <property name="text">
        <string>Exception Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
